docs/_snippets/address.mdx

```diff
@@ -20,7 +20,7 @@
     </ParamField>
 
     <ParamField body="country" type="string" required>
-      Must be "USA".
+      If specified must be "USA", otherwise will default to "USA"
```

Comment:
nit: punctuation - `will default to "USA".`

Reply:
nit: punctuation - `If specified, must be "USA"`

Reply:
lmfao I was thinking this, and was like "aaaahhh fuck it" - good to see we're upholding quality 🚀